memory-store.js

```diff
@@ -6,6 +6,21 @@ var compareTime = require('./compare-time')
  * It is good for tests, but not for server or client usage,
  * because it doesn’t save events to file or localStorage.
  *
+ * Think about this store as a basic store realization.
+ *
+ * Every Store class should provide three standard methods:
+ * add, get and remove.
+ *
+ * Every Store for client should provide two more methods:
+ * getSynced, setSynced.
+ *
+ * @param {object} [options] Store options.
+ * @param {number} [options.synced=0] Events with lower `added` time in current
```

Comment:
In couple swift libraries I found `local`/`remote` prefixes for this case. Also in scuttlebut (https://github.com/dominictarr/scuttlebutt/blob/master/index.js) they used `latest` + `syncSent`/`syncRecv`. I thinking about something like `latestClient`/`latestServer` or `clientSynced`/`serverSynced` as it will be used in client nodes – so we'll always know what is `client` and what is `server` in this case. Also yeah – can work `localSynced`/`remoteSynced`